CMakeLists.txt

```
@@ -340,9 +340,9 @@ if(SDL3MIXER_SNDFILE)
         message(FATAL_ERROR "libsndfile is not vendored.")
     else()
         message(STATUS "Using system libsndfile")
-        find_package(sndfile REQUIRED)
+        find_package(SndFile REQUIRED)
         if(NOT SDL3MIXER_SNDFILE_SHARED)
-            list(APPEND PC_REQUIRES sndfile)
+            list(APPEND PC_REQUIRES SndFile)
```

This is the name of the pkg-config file, which is `sndfile`.
```suggestion
            list(APPEND PC_REQUIRES sndfile)
```